### PR TITLE
Fix layer_norm flatten issue

### PR DESCRIPTION
**Pull request by baojun-nervana**

### CODE REVIEW

File: test/backend/layer_norm.in.cpp [hunk 1]
```diff
@@ -194,3 +194,52 @@ NGRAPH_TEST(${BACKEND_NAME}, layer_norm_bprop_affine)
...
+    auto backend = runtime::Backend::create("${BACKEND_NAME}");
+
+    // Create tensors for input
+    auto data = backend->create_tensor(element::f32, Shape{2, 3, 4, 5});
+    auto delta = backend->create_tensor(element::f32, Shape{2, 3, 4, 5});
+    auto mean = backend->create_tensor(element::f32, Shape{2});
+    auto variance = backend->create_tensor(element::f32, Shape{2});
+    auto scale = backend->create_tensor(element::f32, Shape{60});
+    // Fill in input tensors
+    vector<float> d_input(1, 2 * 3 * 4 * 5);
```
Comment (rkimballn1) [inline test/backend/layer_norm.in.cpp]: This creates a tensor of size 1 and initializes it with the value 120. The size should be 120 elements, not 1.
```suggestion
    vector<float> d_input(2 * 3 * 4 * 5, 1);
```

File: test/backend/layer_norm.in.cpp [hunk 2]
```diff
@@ -194,3 +194,52 @@ NGRAPH_TEST(${BACKEND_NAME}, layer_norm_bprop_affine)
...
+    copy_data(data, d_input);
+    vector<float> dt_input(1, 2 * 3 * 4 * 5);
```
Comment (rkimballn1) [inline test/backend/layer_norm.in.cpp]:
```suggestion
    vector<float> dt_input(2 * 3 * 4 * 5, 1);
```

File: test/backend/layer_norm.in.cpp [hunk 3]
```diff
@@ -194,3 +194,52 @@ NGRAPH_TEST(${BACKEND_NAME}, layer_norm_bprop_affine)
...
+    copy_data(delta, dt_input);
+    vector<float> m_input(1, 2);
```
Comment (rkimballn1) [inline test/backend/layer_norm.in.cpp]:
```suggestion
    vector<float> m_input(2, 1);
```

File: test/backend/layer_norm.in.cpp [hunk 4]
```diff
@@ -194,3 +194,52 @@ NGRAPH_TEST(${BACKEND_NAME}, layer_norm_bprop_affine)
...
+    copy_data(mean, m_input);
+    vector<float> v_input(1, 2);
```
Comment (rkimballn1) [inline test/backend/layer_norm.in.cpp]:
```suggestion
    vector<float> v_input(2, 1);
```

File: test/backend/layer_norm.in.cpp [hunk 5]
```diff
@@ -194,3 +194,52 @@ NGRAPH_TEST(${BACKEND_NAME}, layer_norm_bprop_affine)
...
+    copy_data(variance, v_input);
+    vector<float> s_input(1, 60);
```
Comment (rkimballn1) [inline test/backend/layer_norm.in.cpp]:
```suggestion
    vector<float> s_input(60, 1);
```

File: test/backend/layer_norm.in.cpp [hunk 6]
```diff
@@ -194,3 +194,52 @@ NGRAPH_TEST(${BACKEND_NAME}, layer_norm_bprop_affine)
...
+NGRAPH_TEST(${BACKEND_NAME}, layer_norm_bprop_4d_input)
+{
+    auto p_data = make_shared<op::Parameter>(element::f32, Shape{2, 3, 4, 5});
+    auto p_delta = make_shared<op::Parameter>(element::f32, Shape{2, 3, 4, 5});
+    auto p_mean = make_shared<op::Parameter>(element::f32, Shape{2});
+    auto p_variance = make_shared<op::Parameter>(element::f32, Shape{2});
+    auto p_scale = make_shared<op::Parameter>(element::f32, Shape{60});
+    auto lnb = make_shared<op::LayerNormBackprop>(p_data, p_delta, p_mean, p_variance, p_scale);
+
+    auto output_data = lnb->outputs()[0].as_single_output_node(false);
```
Comment (rkimballn1) [inline test/backend/layer_norm.in.cpp]:
```suggestion
    auto output_data = lnb->output(0);
```

File: test/backend/layer_norm.in.cpp [hunk 7]
```diff
@@ -194,3 +194,52 @@ NGRAPH_TEST(${BACKEND_NAME}, layer_norm_bprop_affine)
...
+    auto output_scale = lnb->outputs()[1].as_single_output_node(false);
```
Comment (rkimballn1) [inline test/backend/layer_norm.in.cpp]:
```suggestion
    auto output_scale = lnb->output(1);
```

File: test/backend/layer_norm.in.cpp [hunk 8]
```diff
@@ -194,3 +194,52 @@ NGRAPH_TEST(${BACKEND_NAME}, layer_norm_bprop_affine)
...
+    auto output_bias = lnb->outputs()[2].as_single_output_node(false);
```
Comment (rkimballn1) [inline test/backend/layer_norm.in.cpp]:
```suggestion
    auto output_bias = lnb->output(2);
```

File: test/backend/layer_norm.in.cpp [hunk 9]
```diff
@@ -194,3 +194,52 @@ NGRAPH_TEST(${BACKEND_NAME}, layer_norm_bprop_affine)
...
+
+    // flatten output_scale
+    auto output_scale_shape = output_scale->get_shape();
+    auto flattened_output_scale = make_shared<op::Reshape>(
+        output_scale, get_default_order(output_scale_shape), Shape{shape_size(output_scale_shape)});
+
+    auto f = make_shared<Function>(NodeVector{output_data, flattened_output_scale, output_bias},
```
Comment (rkimballn1) [inline test/backend/layer_norm.in.cpp]:
```suggestion
    auto f = make_shared<Function>(OutputVector{output_data, flattened_output_scale, output_bias},
```

File: test/backend/layer_norm.in.cpp [hunk 10]
```diff
@@ -194,3 +194,52 @@ NGRAPH_TEST(${BACKEND_NAME}, layer_norm_bprop_affine)
...
+    auto p_variance = make_shared<op::Parameter>(element::f32, Shape{2});
+    auto p_scale = make_shared<op::Parameter>(element::f32, Shape{60});
+    auto lnb = make_shared<op::LayerNormBackprop>(p_data, p_delta, p_mean, p_variance, p_scale);
+
+    auto output_data = lnb->outputs()[0].as_single_output_node(false);
+    auto output_scale = lnb->outputs()[1].as_single_output_node(false);
+    auto output_bias = lnb->outputs()[2].as_single_output_node(false);
+
+    // flatten output_scale
+    auto output_scale_shape = output_scale->get_shape();
```
Comment (rkimballn1) [inline test/backend/layer_norm.in.cpp]:
```suggestion
    auto output_scale_shape = output_scale.get_shape();
```

**Comment by baojun-nervana** [comment]
TESTNOW

**Pull request merged**